chain/client/src/client_actor.rs

```diff
@@ -251,6 +251,8 @@ impl Actor for ClientActor {
     type Context = Context<Self>;
 
     fn started(&mut self, ctx: &mut Self::Context) {
+        self.run_flat_storage_creation_step(ctx);
```

Comment:
nit: to make it consistent with the rest of the code, maybe rename this function to `start_flat_storage_creation`